src/web/hello_js/hello_js.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Hello JS
+level: 1
+author: Arve Seljebu
+---
+
+# Introduksjon {.intro}
+I denne oppgaven blir du introdusert for programmeringsspråket JavaScript. Du
+skal gjøre den klassiske oppgaven *Hello world*, med en katt.
+
+![](jsbin.png)
+
+# Steg 1: Bruke JS Bin {.activity}
+Du skal bruke en nettside som heter *JS Bin* for denne oppgaven. Dette fordi
+JS Bin ser likt ut i alle nettlesere, så da er det enklere å følge stegene.
+
+## Sjekkliste {.check}
+
+- Åpne [JS Bin](http://jsbin.com/?js,console) i et eget vindu.
+- Legg merke til at du får to faner; **JavaScript** og **Console**.
+- Koden skrives i **JavaScript**.
+- Og i **Console** vises tekstbeskjeder som blir skrevet ut.
+- Skriv inn denne koden:
```

Comment:
_Skriv inn denne koden i **JavaScript**_  ?? En god ting kan ikke sies for ofte ;)